ui/src/importer/importerTaskDetails/ImportedDocuments.js

```diff
@@ -61,15 +102,122 @@ export class ImportedDocuments extends React.Component {
         });
       } else {
         this.setState({
-          data: response.data,
           isLoading: true,
+          data: response.data,
         });
       }
+      this.calculateMode();
+      this.calculateStatus();
+      this.calculateStatistics();
     } else {
       this.intervalId && clearInterval(this.intervalId);
     }
   };
 
+  calculateMode = () => {
+    const { data } = this.state;
+    let { mode } = this.state;
```

Comment:
```suggestion
    const { mode } = this.state;
```

Reply:
I'm don't think that I can define that variable as const, as I'm assigning to it later in that function.